test/WebJobs.Script.Tests/Configuration/ServerlessSecurityDefenderOptionsSetupTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Threading.Tasks;
+using Az.ServerlessSecurity.Platform;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ServerlessSecurityDefenderOptionsSetupTests
```

Comment:
In your latest revision, I don't see the ServerlessSecurityDefenderEndToEndTests that you had in your previous revisions - where is that test - can't you get it to work?

Reply:
The E2E test works locally but not in the azure-pipeline build, will add it in the next iteratition.